h2o-genmodel/src/main/java/hex/genmodel/algos/drf/DrfMojoModel.java

```diff
@@ -2,25 +2,26 @@
 
 import hex.ModelCategory;
 import hex.genmodel.GenModel;
-import hex.genmodel.PredictContributions;
-import hex.genmodel.PredictContributionsFactory;
 import hex.genmodel.algos.tree.*;
 
-import java.util.ArrayList;
-import java.util.List;
-
 
 /**
  * "Distributed Random Forest" MojoModel
  */
-public final class DrfMojoModel extends SharedTreeMojoModel implements SharedTreeGraphConverter, PredictContributionsFactory {
+public final class DrfMojoModel extends SharedTreeMojoModelWithContributions implements SharedTreeGraphConverter {
     protected boolean _binomial_double_trees;
 
 
     public DrfMojoModel(String[] columns, String[][] domains, String responseColumn) {
         super(columns, domains, responseColumn);
     }
 
+    @Override
+    protected ContributionsPredictor getContributionsPredictor(TreeSHAPPredictor<double[]> treeSHAPPredictor) {
+        return new ContributionsPredictorDRF(treeSHAPPredictor){
```

Comment:
this class is not abstract yet you have an annonymous inner class here

Reply:
Good point. Done.